src/components/modules/subscriptionsBanner/Logo.tsx

```diff
@@ -0,0 +1,15 @@
+import React, { ReactElement } from 'react';
+import guardianLogo from './guardianLogo.png';
```

Comment:
There's definitely a gap in my knowledge here, but in terms serving this component remotely to one of the platforms I'm not sure how an image import like this will be handled in the bundle we build via rollup. @nicl and @jfsoul may be able to clarify as they've both spent time configuring rollup recently.

Reply:
Is the contribution's service server serving static assets such as images? I actually think in these instances where `.png`s have been used I think we should be using `.svg`s which qwould be more performant and would also be inlined in the bundle.

Reply:
No it's not serving static assets currently. Switching to SVGs and inlining into the bundle sounds like that could be a good way to go.

Reply:
For the packshot image we're using an image hosted at `i.guim.co.uk/img/media` which has been put in the Grid, this seems like the quickest way to add a none `svg` image for now.

Reply:
Great points! I spent quite a bit of time trying to get storybook to find/render svgs without success. I concluded that this was a larger issue, possibly to do with rollup, or maybe it's the way this project has been set up. I threw the pngs in there to have something to work with for the raw task of laying this out. I have had quite a few issues with storybook viewports - I've found it tricky to customise the breakpoints and I'm concerned about the fidelity of it due to large borders. I think it may be worth talking to Simon A about this as I know they use storybook for development in Source, and have probably had to overcome these sorts of issues.